doc/gettingstarted.md

```diff
@@ -21,32 +23,31 @@ In this guide we will get everything up and running on your local machine (on `l
 
 ## Dependencies 
 
-#### Certificates
+### CLI and Stack
+
+For **stable** versions, see the [Releases](https://github.com/TheThingsNetwork/lorawan-stack/releases) page on Github.
+
+### Certificates
+
 By default, the Stack requires a `cert.pem` and `key.pem`, in order to to serve content over TLS.
 
-+ To generate testing certificates, you can use the following command. This requires a [Go environment setup](../DEVELOPMENT.md#development-environment).
++ To generate self-signed certificates, you can use the following command. This requires a [Go environment setup](../DEVELOPMENT.md#development-environment).
 
-    ```bash
-    go run $(go env GOROOT)/src/crypto/tls/generate_cert.go -ca -host localhost && chmod 0444 ./key.pem
-    ```
-    
-+ For production certificates we recommend [LetsEncrypt](https://letsencrypt.org/getting-started/).
+```bash
+go run $(go env GOROOT)/src/crypto/tls/generate_cert.go -ca -host localhost && chmod 0444 ./key.pem
+```
     
++ Keep in mind that self-signed certificates are not trusted by browsers and operating systems, and as such they will return a warning regarding this matter. In order to avoid such issues, we recommend [LetsEncrypt](https://letsencrypt.org/getting-started/).
```

Comment:
```suggestion
+ Keep in mind that self-signed certificates are not trusted by browsers and operating systems, and as such they will return a warning regarding this matter. In order to avoid such issues, we recommend [Let's Encrypt](https://letsencrypt.org/getting-started/).
```